chart/hyrax/templates/deployment-worker.yaml

```diff
@@ -33,9 +33,7 @@ spec:
           command:
             - sh
             - -c
-            - db-wait.sh "$DB_HOST:$DB_PORT"
-            - db-wait.sh "$FCREPO_HOST:$FCREPO_PORT"
-            - db-wait.sh "$SOLR_HOST:$SOLR_PORT"
+            - db-migrate-seed.sh
```

Comment:
i see the motivation for this, but am wondering whether this is the right solution?

are there downsides to the workers trying to run migrations and seeds? i think in a normal deploy on a responsive cluster, this is going to lead to many pods trying to run the migrations all at once, is that okay?

Reply:
I see your point. 


I suppose alternatively we update this to something more along the lines of:

```
- db-wait.sh "$DB_HOST:$DB_PORT" bundle exec rake db:abort_if_pending_migrations
```

Reply:
that seems reasonable.

what happens if we just drop the fcrepo and/or solr checks here completely and do nothing else?

Reply:
I took another pass at this with a dedicated script that solely does waiting
now.

Reply:
do you think it would be viable to remove the whole container? i'm noticing that the only DB backend that really ought to matter to `sidekiq` (rather than the individual jobs) isn't actually in this check: `redis`.

Reply:
@mcritchlow and i chatted about this in a back channel and decided that it's *likely* `sidekiq` is going to try to run Rails initializers eagerly, so we'll need these checks.

i'm going to merge, and Matt is going to try a deploy with the whole init container removed to confirm that it is needed.